docs/docs/providers/prompts/remote_mlflow.mdx

```diff
@@ -0,0 +1,852 @@
+---
+description: |
+  [MLflow](https://mlflow.org/) is a remote provider for centralized prompt management and versioning
```

Comment:
This is a great amount of documentation. Arguably too good. Probably we could rely a little more on linking to MLFlow where relevant and keep the llama stack specific documentation. It's not a big deal though and we can keep it in.

Reply:
Simplifying it to 4 lines in the description section.